frontend/javascripts/oxalis/geometries/skeleton.js

```diff
@@ -485,6 +489,36 @@ class Skeleton {
     });
   }
 
+  /**
+   * Updates a node's position and that of its edges in a WebGL buffer.
+   */
+  updateNodePosition(treeId: number, nodeId: number, position: Vector3, tree: Tree) {
+    const bufferNodeId = this.combineIds(nodeId, treeId);
```

Comment:
This might be premature-optimization speaking in me, but I'd feel a bit more comfortable, if this method checked whether the `position` really changed with the `updateAction`. A simple `if (._isEqual(tree.position, position)) return;` should already solve this, in my opinion. Since this code path will be taken quite rarely, this is not really important (at least now), but I'd like to avoid establishing the pattern of doing redundant stuff :)

Reply:
I also thought about this (as I wrote this code), however, I don't think it's easily possible.

> A simple if (._isEqual(tree.position, position)) return; should already solve this, in my opinion.

The tree does not have a position, but each node has one. However, even if we would check the node's position, it would already be the new one, wouldn't it? We would need a reference to the old tracing to check the previous position of the node.

Reply:
Oops, very good point. I glanced over this too quickly. One could look up the old position in the nodeBuffer, but this doesn't appeal to me anymore ^^ Then, half of the function's code is already executed. So, let's leave it as is :)